doc/start/hardware-recommendations.rst

```diff
@@ -584,36 +600,38 @@ found above and elsewhere within this documentation.
 |              |                |   Benchmarking is highly recommended.   |
 |              +----------------+-----------------------------------------+
 |              | RAM            | - 4GB+ per daemon (more is better)      |
-|              |                | - 2-4GB may function but may be slow    |
+|              |                | - 2-4GB may function will be slow       |
 |              |                | - Less than 2GB is not recommended      |
 |              +----------------+-----------------------------------------+
-|              | Storage Drives |  1x storage drive per OSD               |
+|              | Storage Drives | 1x storage drive per OSD in most cases. |
+|              |                | PCIe Gen 5+ SSDs larger than 30TB may   |
+|              |                | benefit from being split into two or    |
+|              |                | OSDs.                                   |
 |              +----------------+-----------------------------------------+
-|              | DB/WAL         |  1x SSD partion per HDD OSD             |
+|              | DB/WAL offload |  1x SSD partion per HDD OSD             |
 |              | (optional)     |  4-5x HDD OSDs per DB/WAL SATA SSD      |
-|              |                |  <= 10 HDD OSDss per DB/WAL NVMe SSD    |
+|              |                |  <= 15 HDD OSDss per DB/WAL NVMe SSD    |
 |              +----------------+-----------------------------------------+
-|              | Network        |  1x 1Gb/s (bonded 10+ Gb/s recommended) |
+|              | Network        |  1x 1Gb/s (bonded 25+ Gb/s recommended) |
```

Comment:
I'd recommend a space between the number and unit but I see there's no more space in the column.

Maybe the leading double space in all the cells in this column could be changed to a single space to accommodate.

I can address in a separate PR if needed.